activiti-cloud-runtime-bundle-service/activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-impl/src/main/java/org/activiti/cloud/services/rest/controllers/ProcessInstanceAdminControllerImpl.java

```diff
@@ -56,98 +63,124 @@
 @RestController
 public class ProcessInstanceAdminControllerImpl implements ProcessInstanceAdminController {
 
-    private final ProcessInstanceRepresentationModelAssembler representationModelAssembler;
+  private static final Logger LOGGER = LoggerFactory.getLogger(ProcessInstanceAdminControllerImpl.class);
 
-    private final AlfrescoPagedModelAssembler<ProcessInstance> pagedCollectionModelAssembler;
+  private final Set<ProcessInstanceStatus> deleteStatuses = Set.of(ProcessInstanceStatus.COMPLETED, ProcessInstanceStatus.CANCELLED);
 
-    private final ProcessAdminRuntime processAdminRuntime;
+  private final String DELETE_PROCESS_NOT_ALLOWED = "Process Instance %s is not in status: " +
+      String.join(", ", deleteStatuses.stream().map(Enum::name).collect(Collectors.toList()));
 
-    private final SpringPageConverter pageConverter;
+  private final ProcessInstanceRepresentationModelAssembler representationModelAssembler;
 
-    private final ProcessVariablesPayloadConverter variablesPayloadConverter;
```

Comment:
Please, check your IDE settings. It seems you're using indentation with 2 spaces instead of 4.

Reply:
Done, thank you.